superset-frontend/src/components/Datasource/CollectionTable.tsx

```diff
@@ -222,6 +226,11 @@ export default class CRUDCollection extends React.PureComponent<
     return label;
   }
 
+  getTooltip(col: any) {
```

Comment:
can we put a type for col here too? It looks like in line 232 it's defined as a string.

Reply:
```suggestion
  getTooltip(col: string) {
```